tests/integration_tests/functional/test_drives.py

```diff
@@ -319,3 +366,21 @@ def _check_drives(test_microvm, assert_dict, keys_array):
         stdout.read(),
         assert_dict,
         keys_array)
+
+
+def _add_block_device(
```

Comment:
Can we use Microvm.add_drive() instead of duplicating this here ?

Reply:
Yup, I somehow missed that method 🙈